proving_system/stark/src/fri/mod.rs

```diff
@@ -97,7 +98,8 @@ where
         degree = p_i.degree();
 
         last_poly = p_i.clone();
-        last_coef = last_poly.coefficients.get(0).unwrap();
+        last_coef = last_poly.evaluate(&zero);
```

Comment:
This change is equal to `unwrap_or(FieldElement::zero())`, I think it may just be faster to do that, instead of evaluating the polynomial, but this is alright too

Reply:
Addressed in [897b08b](https://github.com/lambdaclass/lambdaworks/pull/232/commits/897b08b8a841ff23d46e543a896fe06636c6f215)